includes/components/class-admin.php

```diff
@@ -1481,6 +1484,40 @@ public function enqueue_scripts() {
 		if ( in_array( $pagenow, [ 'edit-tags.php', 'term.php' ], true ) ) {
 			wp_enqueue_media();
 		}
+
+		// Get pointers.
+		$pointers = apply_filters( 'hivepress/v1/admin/tooltips', [] );
+
+		if ( ! $pointers ) {
+			return;
+		}
+
+		// Get dismissed pointers.
+		$dismissed = (array) get_user_meta( get_current_user_id(), 'hp_dismissed_pointers', true );
+
+		// Set valid pointers.
+		$valid_pointers = [];
+
+		// Check pointers.
+		foreach ( $pointers as $pointer_id => $pointer ) {
+			if ( ! $pointer_id || in_array( $pointer_id, $dismissed ) || ! $pointer || ! $pointer['target'] || ! $pointer['options'] ) {
```

Comment:
if some properties are optional it's better to fill the defaults after apply_filters or here in the loop instead of requiring them, or all of these are required?